src/components/students/studentDetail/studentProfile/studentProfileCards/editStudentProfile/EditStudentProfile.tsx

```diff
@@ -72,14 +72,13 @@ export const EditStudentProfile: React.FC<EditStudentProfileProps> = ({
                 </div>
                 <div className="w-full h-full ">
                     <div className="flex flex-col h-[20%] justify-evenly">
-                        <div className="flex">
-                            <h1 className="text-[26px] w-[162px] h-[34px] my-0 leading-[34px] font-bold text-[rgba(40,40,40,1)]">
-                                Editar datos
-                            </h1>
-                        </div>
-                        <div className="flex justify-between pr-4">
-                            <div className="w-[86px] h-[77px] border border-[rgba(128,128,128,1)] rounded-2xl ">
+                        <h1 className="text-2xl my-0 mb-2 font-bold text-[rgba(40,40,40,1)]">
+                            Editar datos
+                        </h1>
+                        <div className="flex  justify-between pr-4 bg-green-200 mb-2">
```

Comment:
Should this div have a green background?

Reply:
Nooo ! se me pasó , gracias.